Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -1245,3 +1254,20 @@ theorem comap_map_eq_self_of_injective
 end Subalgebra
 
 end MapComap
+
+variable {R A : Type*} [CommSemiring R] [Semiring A] [Algebra R A]
+
+/-- Turn a non-unital subalgebra containing `1` into a subalgebra. -/
+def NonUnitalSubalgebra.toSubalgebra (S : NonUnitalSubalgebra R A) (h1 : (1 : A) ∈ S) :
+    Subalgebra R A :=
```

Comment:
nit: could switch to `where` notation now this is moving anyway.

Reply:
Well spotted. I had meant to do that